election/packages/ge2020/elections/index.js

```diff
@@ -1,12 +1,11 @@
 const axios = require('axios')
 const google_api_token = process.env.GOOGLE_CIVIC_API_TOKEN
-const path = `https://www.googleapis.com/civicinfo/v2/elections?key=${google_api_token}`
 
 async function main(args) {
     return axios
-        .get(path)
+        .get(`https://www.googleapis.com/civicinfo/v2/elections?key=${google_api_token}`)
```

Comment:
if you leave `path` as a constant then don't need to string interpolate on every call.